modules/dag-l1/src/main/scala/org/tessellation/dag/l1/infrastructure/block/BlockService.scala

```diff
@@ -0,0 +1,222 @@
+package org.tessellation.dag.l1.infrastructure.block
```

Comment:
service should be in the domain

Reply:
The `BlockService` trait is in the domain. Should I also move this concrete implementation to the domain?